features/karaf/src/main/feature/feature.xml

```diff
@@ -27,6 +27,7 @@
     <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.console/${project.version}</bundle>
     <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.monitor/${project.version}</bundle>
     <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.voice/${project.version}</bundle>
+	<bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.audio/${project.version}</bundle>
```

Comment:
The whole file should use spaces for indentation. So please keep the current scheme.